megamek/src/megamek/client/ui/swing/PhysicalDisplay.java

```diff
@@ -1477,16 +1477,10 @@ public void hexMoused(BoardViewEvent b) {
         if ((b.getModifiers() & InputEvent.CTRL_DOWN_MASK) != 0) {
             return;
         }
-        if (clientgui.getClient().isMyTurn()
```

Comment:
Why remove this check? I did note it wasn't part of the other ones though

Reply:
Restored.